apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data.Tests/TimestampTest.cs

```diff
@@ -151,7 +151,7 @@ public void FromProtoOrNull_Null()
         [Fact]
         public void FromProtoOrNull_NonNull()
         {
-            var proto = new wkt::Timestamp { Seconds = 1, Nanos = 100 };
+            var proto = CreateProtoTimestamp(1, 500);
```

Comment:
...likewise ;)

Reply:
Whoops - those are probably the source of the test failures. Will fix.

Reply:
Fixed. (I could have sworn I'd replied to these, but github seems to have eaten my comments...)